### PR TITLE
fix Mac OS -> macOS

### PR DESCRIPTION
**Pull request by shcheklein**
Thanks @wbednarski for noticing this

### CODE REVIEW

File: content/docs/sidebar.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "source": "install/index.md",
     "children": [
       {
-        "label": "Mac OS",
+        "label": "macOS",
```
Comment (jorgeorpinel) [inline content/docs/sidebar.json:13]:
```suggestion
        "label": "MacOS",
```

File: src/components/DownloadButton/index.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ const itemsByOs = {
     download: false
   },
   [OS.OSX]: {
-    title: 'Mac OS',
+    title: 'macOS',
```
Comment (jorgeorpinel) [inline src/components/DownloadButton/index.tsx:29]:
```suggestion
    title: 'MacOS',
```

**Review by jorgeorpinel — APPROVED** [review]
"MacOS" is also correct, I think. `macOS` is more of a logo. Formally speaking any proper name starts with a capital letter and in lists like "Linux, Windows, MacOS" looks more consistent (reason why we had it that way). But any way is fine probably, as long as we use the same form everywhere.

In case we roll back I'd just make these 2 changes below (removing the space):

File: content/docs/sidebar.json (hunk 1 above)
Comment (shcheklein) [reply content/docs/sidebar.json:13]: macOS is the official name for the OS (like FreeBSD would not be Freebsd for consistency). I think logo being the same is just a coincidence. 

> Formally speaking any proper name starts with a capital letter

brand names are written the way the are written I think. We can refer to the Wiki page - https://en.wikipedia.org/wiki/MacOS - they use macOS everywhere.

Comment (jorgeorpinel) [reply content/docs/sidebar.json:13]: Sent from an Iphone 😋

**Pull request merged**